client/my-sites/post-type-list/post-actions-ellipsis-menu/duplicate.jsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import PopoverMenuItem from 'components/popover/menu-item';
+import QueryPostTypes from 'components/data/query-post-types';
+import { mc } from 'lib/analytics';
+import { canCurrentUser } from 'state/selectors';
+import { getPost } from 'state/posts/selectors';
+import { getPostType } from 'state/post-types/selectors';
+import { getCurrentUserId, isValidCapability } from 'state/current-user/selectors';
+import { getEditorDuplicatePostPath } from 'state/ui/editor/selectors';
+import config from 'config';
+
+function PostActionsEllipsisMenuDuplicate( { translate, siteId, canEdit, duplicateUrl, isKnownType } ) {
+	if ( ! config.isEnabled( 'posts/post-type-list' ) || ! canEdit ) {
+		return null;
+	}
+
+	function bumpStat() {
+		mc.bumpStat( 'calypso_cpt_actions', 'duplicate' );
+	}
+
+	return (
+		<PopoverMenuItem href={ duplicateUrl } onClick={ bumpStat } icon="pages">
+			{ siteId && ! isKnownType && <QueryPostTypes siteId={ siteId } /> }
+			{ translate( 'Duplicate', { context: 'verb' } ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Skipping duplicate' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.